modules/aruco/tutorials/aruco_board_detection/aruco_board_detection.markdown

```diff
@@ -198,7 +198,9 @@ Finally, a full example of board detection:
 
 Sample video:
 
-[![Board Detection video](http://img.youtube.com/vi/Q1HlJEjW_j0/0.jpg)](https://youtu.be/Q1HlJEjW_j0)
+@htmlonly
+<iframe width="420" height="315" src="https://www.youtube.com/embed/Q1HlJEjW_j0" frameborder="0" allowfullscreen></iframe>
+@endhtmlonly
```

Comment:
No, this instruction does not work in doxygen - https://www.stack.nl/~dimitri/doxygen/manual/markdown.html